docs/DEVELOPING.md

```diff
@@ -111,9 +128,38 @@ To sync both sides, the CLI uses these steps:
   - if not, adds it as an `upsert` operation into a `bulk` call to Elasticsearch
 - for each id from Elasticsearch that is not present in the documents sent by the data source class, adds it as a `delete` operation into the `bulk` call
 - `bulk` calls are emitted every 500 operations (this is configurable for slow networks).
+- If incremental sync is available for the connector, `self._sync_cursor` should also be updated in the connector document in the end, so that when an incremental sync starts, it knows where to start from.
```

Comment:
```suggestion
- If incremental sync is available for the connector, `self._sync_cursor` should also be updated in the connector document in the end of the sync, so that when an incremental sync starts, it knows where to start from.
```